cmd/boost/deal_status_cmd.go

```diff
@@ -138,6 +138,9 @@ func statusMessage(resp *types.DealStatusResponse) string {
 	case dealcheckpoints.IndexedAndAnnounced.String():
 		return "Sealing"
 	case dealcheckpoints.Complete.String():
+		if resp.DealStatus.Error != "" {
```

Comment:
Not sure why we have both `resp.Error` and `resp.DealStatus.Error`.

We are handling the first above, and now here the second error...

Reply:
The difference is that resp.Error is if there was an error with processing the deal status request.
resp.DealStatus.Error is if the deal errored out.